### PR TITLE
add small section on prefer-reduced-motion

### PR DESCRIPTION
**Pull request by colorful-tones**
This addresses #105 

Feedback is welcome. Thanks!

### CODE REVIEW

File: _includes/markdown/CSS.md [hunk 1]
```diff
@@ -278,6 +278,26 @@ Articles worth reading:
 * [CSS vs JavaScript Animations](https://developers.google.com/web/fundamentals/look-and-feel/animations/css-vs-javascript?hl=en)
 * [requestAnimationFrame](https://developer.mozilla.org/en-US/docs/Web/API/window/requestAnimationFrame)
 
+<h2 id="accessibility" class="anchor-heading">Accessibility {% include Util/top %}</h2>
+
+Not every animation brings pleasure to the end user. In some cases motion can trigger harmful reactions from users with vestibular disorders, epilepsy or even migraines.
+
+While the `prefer-reduced-motion` CSS media feature does not have the widest support yet (mostly just Safari and Firefox). We still recommend applying `prefer-reduced-motion`, as it is simple to implement and affords a better experience for those using supported browsers.
```
Comment (JodiWarren) [inline _includes/markdown/CSS.md]: Some very small grammatical and content suggestions:

- Remove "While"
- Prepend "We" with "However, "
- Remove "mostly" - no harm in being specific!
- Replace `yet` with current date, as support may change. You could also insert a link to caniuse: https://caniuse.com/#feat=prefers-reduced-motion.

**Review by JodiWarren — CHANGES_REQUESTED** [review]
Some very small suggestions. Good update overall!

File: _includes/markdown/CSS.md (hunk 1 above)
Comment (colorful-tones) [reply _includes/markdown/CSS.md]: Great suggestions and I applied all. I did a bit of a deviation on the last bullet though.

Comment (samikeijonen) [reply _includes/markdown/CSS.md]: If I remember correctly Chrome is adding support in the next release or something.

**Pull request merged**